JavaAssignment/src/piyushS/Q10.java

```diff
@@ -0,0 +1,31 @@
+package piyushS;
+
+public class Q10 {
+	static int sum = 0;
+
+	static void isPrime(int number) {
```

Comment:
The method name is good, but need to add comments before your code, method and if required before loops or condition.